### ISSUE TITLE
Implement MAE Loss with tensorflow and pytorch 

### ISSUE THREAD

**Issue by pavanreddyml** [pavanreddyml/adversarial-lab#10]
Implement a Mean AbsoluteError Loss with tf and torch similar to 'adversarial_lab/core/losses/categorical_cross_entropy.py'. The losses must be compatible with autograd and autodiff
